src/Common/Epoll.cpp

```diff
@@ -70,6 +70,9 @@ size_t Epoll::getManyReady(int max_events, epoll_event * events_out, bool blocki
 
         if (ready_size == -1 && errno != EINTR)
             throwFromErrno("Error in epoll_wait", DB::ErrorCodes::EPOLL_ERROR);
+        
+        if (errno == EINTR)
+            LOG_DEBUG(&Poco::Logger::get("Epoll"), "EINTR");
```

Comment:
Maybe `LOG_TEST` is better for this change?

```suggestion
            LOG_TEST(&Poco::Logger::get("Epoll"), "EINTR");
```

Reply:
I am not sure that I want to merge this PR (because it can lead to mess in the log), just wanted to look at CI in this PR and restart tests sometimes in attempt to get hung stress test.

Reply:
I think it's ok to merge it with `LOG_TEST`, because `Test` log level should not be used in production and it needed mostly for debugging obscure failures in CI.